test/adjoint/test_fp_iteration.py

```diff
@@ -98,7 +98,16 @@ def test_noconvergence(self):
     def test_no_late_convergence(self):
         self.parameters.drop_out_converged = True
         mesh_seq = self.mesh_seq(time_partition=TimePartition(1.0, 2, [0.5, 0.5], []))
-        mesh_seq.fixed_point_iteration(oscillating_adaptor0, parameters=self.parameters)
+        solver_kwargs = (
+            {"track_coefficients": False}
+            if type(mesh_seq) is GoalOrientedMeshSeq
+            else {}
+        )
+        mesh_seq.fixed_point_iteration(
+            oscillating_adaptor0,
+            parameters=self.parameters,
+            solver_kwargs=solver_kwargs,
+        )
```

Comment:
This was necessary to add since these tests use dummy `get_solver` functions, so there is no form to get the coefficients from

Reply:
No longer relevant